### PR TITLE
olya_task13_list_with_unique_items

### PR DESCRIPTION
**Pull request by olyapetryshyn**

### CODE REVIEW

File: olya_petryshyn/task0013_list_with_unique_items.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+list_of_ints = [12, 24, 35, 24, 88, 120, 155, 88, 120, 155]
+
+def unique_list_reversed(some_list):
+    new_set = set()
+    for item in some_list:
+        new_set.add(item)
+    new_list = list()
```
Comment (pivancy) [inline olya_petryshyn/task0013_list_with_unique_items.py]: The general idea is good, but why don't you convert set into the list?

It should work like this:

`return list(new_set)[::-1]`

Don't you find it easier?

Comment (olyapetryshyn) [reply olya_petryshyn/task0013_list_with_unique_items.py]: Yeah, also works. Fixed.

File: olya_petryshyn/task0013_list_with_unique_items.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+list_of_ints = [12, 24, 35, 24, 88, 120, 155, 88, 120, 155]
+
+def unique_list_reversed(some_list):
+    new_set = set()
+    for item in some_list:
+        new_set.add(item)
+    return list(new_set)[::-1]
```
Comment (pivancy) [inline olya_petryshyn/task0013_list_with_unique_items.py]: You can also simplify this in this way:

    def unique_list_reversed(some_list):
        return list(set(some_list))[::-1]

Comment (olyapetryshyn) [reply olya_petryshyn/task0013_list_with_unique_items.py]: Done.

**Comment by pivancy** [comment]
👍

**Pull request merged**